### PR TITLE
Add project structure according to cosmic python

### PR DESCRIPTION
**Pull request by cosletmihai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
-# lam4doc
+# LAM 4 DOC
 Initiative for Modeling the Legal Analysis Methodology (LAM): Document generation service
```
Comment (costezki) [inline README.md:2]: nice! we have a readme from the beginning :)

File: docker/.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
+LAM_API_LOCATION=http://lam-api
+LAM_API_PORT=5000
```
Comment (costezki) [inline docker/.env:2]: LAM service | 8050 | 4050 |   |   | lam-data
-- | -- | -- | -- | -- | --

File: docker/api/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+COPY . /usr/src/app
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+
+CMD gunicorn -w 2 --bind 0.0.0.0:${LAM_API_PORT} --reload --log-level=debug lam4doc.entrypoints.api.run:app
```
Comment (costezki) [inline docker/api/Dockerfile:13]: nice, already set the app module

File: lam4doc/entrypoints/api/handlers.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# Date:  03/11/2020
+# Author: Mihai Coșleț
+# Email: coslet.mihai@gmail.com
+
+"""
+OpenAPI method handlers.
+"""
+
+
+def lam_example():
```
Comment (costezki) [inline lam4doc/entrypoints/api/handlers.py:13]: :)

File: requirements/common.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+requests==2.24.0  # https://requests.readthedocs.io/en/master/
+connexion[swagger-ui]==2.7.0  # https://github.com/zalando/connexion
+Flask==1.1.2  # https://flask.palletsprojects.com/en/1.1.x/e
+Flask-Wtf==0.14.3  # https://flask-wtf.readthedocs.io/en/stable/
+
+selenium==3.141.0  # https://selenium-python.readthedocs.io/
```
Comment (costezki) [inline requirements/common.txt:6]: I believe selenium is for testing alone, so probably shall be in the -dev requirememnts

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,19 @@
+[tool:pytest]
+addopts =
+    --cov=lam4doc
```
Comment (costezki) [inline setup.cfg:3]: nice, you paid attention to everything!

**Review by costezki — APPROVED** [review]
very well done! 
fast and accurate!

File: setup.cfg (hunk 6 above)
Comment (cosletmihai) [reply setup.cfg:3]: yes, of course :100:

**Review by costezki — COMMENTED** [review]
well done

**Pull request merged**